src/auth-service/utils/mailer.js

```diff
@@ -543,12 +519,6 @@ const mailer = {
     next
   ) => {
     try {
-      const checkResult = await SubscriptionModel(
```

Comment:
Hi @BenjaminSsempala , could you shed some light as to why these notification status checks are being removed? Could you share a small high level write up of this approach so that I can easily review this PR? I am a little bit confused to be honest. :)

Reply:
@Baalmart I removed the subscription checks for emails, that we might consider compulsory like forgot password, getting token, verifying email etc

Reply:
Okay, thanks @BenjaminSsempala , this makes a lot of sense! Thanks!